src/identity/hcs/did/event/hcs-did-verification-relationship-event.ts

```diff
@@ -0,0 +1,75 @@
+import { PublicKey } from "@hashgraph/sdk";
+import { Hashing } from "../../../..";
+import { HcsDidEvent } from "./hcs-did-event";
+import { HcsDidEventName } from "./hcs-did-event-name";
+
+export type VerificationRelationshipType =
+    | "authentication"
+    | "assertionMethod"
+    | "keyAgreement"
+    | "capabilityInvocation"
+    | "capabilityDelegation";
+
+export class HcsDidVerificationRelationshipEvent extends HcsDidEvent {
+    public static KEY_TYPE = "Ed25519VerificationKey2018";
+
+    protected readonly name = HcsDidEventName.VERIFICATION_RELATIONSHIP;
+
+    protected id: string;
+    protected type = HcsDidVerificationRelationshipEvent.KEY_TYPE;
+    protected relationshipType: VerificationRelationshipType;
+    protected controller: string;
+    protected publicKey: PublicKey;
+
+    /**
+     * TODO: I guess controller param is not necessary and can be derived from the publicKey, right?
```

Comment:
Not sure, How do we make sure whoever tries to construct this event is the current owner of provided DID / public key?
the only way to identify is, by looking at verificationMethod and making sure the Id of #did-root-key is the same as DID.